--- a/target/linux/mediatek/dts/mt7622-netgear-wax206.dts
+++ b/target/linux/mediatek/dts/mt7622-netgear-wax206.dts
@@ -102,9 +102,9 @@
 				read-only;
 			};
 
-			partition@2000000 {
+			partition@4000000 {
 				label = "ubi";
-				reg = <0x2000000 0x2000000>;
+				reg = <0x4000000 0xc000000>;
 			};
 		};
 	};
--- a/target/linux/mediatek/image/mt7622.mk
+++ b/target/linux/mediatek/image/mt7622.mk
@@ -211,6 +211,7 @@ define Device/netgear_wax206
   DEVICE_MODEL := WAX206
   DEVICE_PACKAGES := kmod-mt7915e kmod-mt7915-firmware
   SUPPORTED_DEVICES += netgear,wax206
+  UBI_ROOT_SIZE := 160MiB
 endef
 TARGET_DEVICES += netgear_wax206
